drivers/sensor/ite/tach_ite_it8xxx2: don't clear both status at once

RuibinChang opened the pull request:
When two tach use simultaneously, we find that both tach status will be cleared at once, which causes one of tach not to get r.p.m. 
So I change it to clear status only one tach at a time, and also run clang format.

This issue is reported by Huaqin:
https://partnerissuetracker.corp.google.com/issues/404067663?pli=1

ubieda reviewed (changes requested):
Please don't mix formatting changes with functional changes. If reformatting is desired, please split in two patches.

RuibinChang committed:
drivers/sensor/ite/tach_ite_it8xxx2: don't clear both tach status at once

When two tach use simultaneously, I find that the both tach status will
be cleared at once, which causes one of tach not to get rpm speed.
So I change it to clear status only one tach at a time.

This issue is reported by Huaqin:
https://partnerissuetracker.corp.google.com/issues/404067663?pli=1

Signed-off-by: Ruibin Chang <Ruibin.Chang@ite.com.tw>

RuibinChang commented:
> Please don't mix formatting changes with functional changes. If reformatting is desired, please split in two patches.

Done, split into two commits.

RuibinChang committed:
drivers/sensor/ite/tach_ite_it8xxx2: run clang format

Run clang format.

Signed-off-by: Ruibin Chang <Ruibin.Chang@ite.com.tw>

RuibinChang commented:
use existing macro `IT8XXX2_PWM_T0DVS` and `IT8XXX2_PWM_T1DVS`
